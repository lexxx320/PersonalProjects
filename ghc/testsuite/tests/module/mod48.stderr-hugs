ERROR "mod48.hs":5 - Pattern binding illegal in class declaration
