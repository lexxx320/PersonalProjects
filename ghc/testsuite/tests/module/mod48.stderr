
mod48.hs:5:3:
    Pattern bindings (except simple variables) not allowed in instance declarations
	(x, y) = error "foo"
